### PR TITLE
Add new Code Signing Wizard 

### PR DESCRIPTION
**Pull request by doomsayer13**

### CODE REVIEW

File: source/_global_site_tag_with_optimize.slim [hunk 1]
```diff
@@ -0,0 +1,7 @@
+javascript:
+    window.dataLayer = window.dataLayer || [];
+    function gtag(){
+        dataLayer.push(arguments);
+    }
+    gtag('js', new Date());
+    gtag('config', 'UA-53276411-11', { 'optimize_id': 'GTM-M5BNCWH'});
```
Comment (gaborszakacs) [inline source/_global_site_tag_with_optimize.slim]: Pls consider moving these (and similar keys) into a config file.

File: source/layouts/layout.html.slim [hunk 2]
```diff
@@ -14,4 +14,6 @@ html
 		== favicon_tag "apple-touch-icon-120x120.png", href: image_path("apple-touch-icon-120x120.png"), rel: 'apple-touch-icon', type: 'image/png', sizes: "120x120"
 		== favicon_tag "apple-touch-icon-152x152.png", href: image_path("apple-touch-icon-152x152.png"), rel: 'apple-touch-icon', type: 'image/png', sizes: "152x152"
 	body[ng-app="BitriseWorkflowEditor" ng-cloak]
+		- if build?
```
Comment (gaborszakacs) [inline source/layouts/layout.html.slim]: This is a bit confusing, should it differentiate between local and website mode?

**Comment by doomsayer13** [comment]
@erosdome @gaborszakacs What do you think do we still need this PR or I can close it?

**Comment by gaborszakacs** [comment]
> @erosdome @gaborszakacs What do you think do we still need this PR or I can close it?

Not sure what was this about or why did it get stuck :) We can close it I guess.

**Pull request closed without merging**